src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,12 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+#### Asking filter questions to simplify a complex question
+
+A series of simple questions can be easier to answer than one complex question. Especially if parts of it aren’t relevant to all users. A 'filter' question will let the user skip parts of a question that do not apply to them so they do not get confused.
```

Comment:
Bit of a nitpick, but in the other change 'filter question' is in quotes while here it is "'filter' question"; does this inconsistency matter?

Reply:
Good catch! It's probably should be 'filter question', as it's the term that's misunderstood rather than just the word filter.

Reply:
```suggestion
A series of simple questions can be easier to answer than one complex question. Especially if parts of it aren’t relevant to all users. A 'filter question' will let the user skip parts of a question that do not apply to them so they do not get confused.
```